rdkit/Chem/Randomize.py

```diff
@@ -43,6 +43,17 @@ def RandomizeMolBlock(molB):
     inL = '% 3d% 3d' % (idx1 + 1, idx2 + 1) + inL[6:]
     res.append(inL)
     idx += 1
+  #Charges
+  for i in range(idx, len(splitB)):
+    if splitB[i][0:6] == "M  CHG":
+      line = splitB[i]
+      chargeline = line.split()
+      col = line[0:9]
+      for i in range(3,len(chargeline),2):
+        col = col +"%4i%4i"%(order.index(int(chargeline[i])-1)+1,int(chargeline[i+1])+1)
```

Comment:
Please use spaces around operators as in the rest of the code.

If you use this helper function:
```
def grouped(iterable, n):
  return zip(*[iter(iterable)]*n)
```
you can rewrite the code in a form that may be easier to understand:
```
for atom, charge in grouped(chargeline[3:], 2):
  atom = order.index(int(atom) - 1) + 1
  charge = int(charge) + 1
  col += "%4i%4i" % (atom, charge)
```
I don't understand why you add 1 to the charge (or do I missunderstand)? If you write tests, you should also have a molblock for a negatively charged molecule.